### PR TITLE
redemption rollback - v1.10.0

### PR DESCRIPTION
**Pull request by LukaszWiktor**

### CODE REVIEW

File: voucherify.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var util = require("util");
 var request = require("request");
 var when = require("when");
 
-var backendUrl = "https://api.voucherify.io/v1";
+var backendUrl = "http://localhost:8080/v1";
```
Comment (tpindel) [inline voucherify.js]: ???

Comment (LukaszWiktor) [inline voucherify.js]: fixed

File: README.md [hunk 2]
```diff
@@ -463,6 +491,84 @@ voucherify.redemptions(filter)
...
+       .catch(function (error) {
+           console.error("Error: %s", error);
+       });
+```
+
+Result:
+```
+{
+    "id": "rr_1634wLkb8glgRXrTmsxRzDBd",
+    "object": "rollback",
```
Comment (tpindel) [inline README.md]: @LukaszWiktor I'd change the name of object to `redemption_rollback`.

Comment (LukaszWiktor) [inline README.md]: @tpindel Changed.

**Pull request merged**